frontend/src/pages/FunctionalComponent/RecurringRunDetailsV2FC.tsx

```diff
@@ -0,0 +1,207 @@
+            (errorMessage ? ' Click Details for more information.' : ''),
+          mode: 'warning',
+        });
+        return;
+      }
+    },
+    { enabled: !!experimentId, staleTime: 0 },
+  );
+
+  useEffect(() => {
+    setToolbarState(getInitialToolbarState());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [recurringRun]);
+
+  useEffect(() => {
+    if (toolbarState) {
+      toolbarState.actions[ButtonKeys.ENABLE_RECURRING_RUN].disabled =
+        recurringRun?.status === V2beta1RecurringRunStatus.ENABLED;
+      toolbarState.actions[ButtonKeys.DISABLE_RECURRING_RUN].disabled =
+        recurringRun?.status !== V2beta1RecurringRunStatus.ENABLED;
+      toolbarState.pageTitle = recurringRun?.display_name!;
+      toolbarState.breadcrumbs = getBreadcrumbs(experiment);
+      updateToolbar(toolbarState);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [toolbarState, experiment, recurringRun]);
+
+  useEffect(() => {
+    refetchRecurringRun();
+  }, [refresh, refetchRecurringRun]);
+
+  const getInitialToolbarState = (): ToolbarProps => {
+    const buttons = new Buttons(props, Refresh);
+    return {
+      actions: buttons
+        .cloneRecurringRun(() => (recurringRun ? [recurringRun.recurring_run_id!] : []), true)
+        .refresh(Refresh)
+        .enableRecurringRun(() => (recurringRun ? recurringRun.recurring_run_id! : ''))
+        .disableRecurringRun(() => (recurringRun ? recurringRun.recurring_run_id! : ''))
+        .delete(
+          () => (recurringRun ? [recurringRun.recurring_run_id!] : []),
+          'recurring run config',
+          deleteCallback,
+          true /* useCurrentResource */,
+        )
+        .getToolbarActionMap(),
+      breadcrumbs: [],
+      pageTitle: '',
+    };
+  };
+
+  const deleteCallback = (_: string[], success: boolean) => {
+    if (success) {
+      const breadcrumbs = props.toolbarProps.breadcrumbs;
+      const previousPage = breadcrumbs.length
+        ? breadcrumbs[breadcrumbs.length - 1].href
+        : RoutePage.EXPERIMENTS;
+      props.history.push(previousPage);
+    }
+  };
+
+  return (
+    <div className={classes(commonCss.page, padding(20, 'lr'))}>
+      {recurringRun && (
+        <div className={commonCss.scrollContainer}>
+          <div className={padding(20)}>
+            <DetailsTable
+              title='Recurring run details'
+              fields={getRecurringRunDetails(recurringRun)}
+            ></DetailsTable>
+            <DetailsTable title='Run triggers' fields={getRunTriggers(recurringRun)}></DetailsTable>
+            <DetailsTable
+              title='Run parameters'
+              fields={getRunParameters(recurringRun)}
+            ></DetailsTable>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function getBreadcrumbs(experiment?: V2beta1Experiment): Breadcrumb[] {
+  const breadcrumbs: Breadcrumb[] = [];
+  if (experiment) {
```

Comment:
Can we confirm the experimentId also exists as well? It is to avoid failing without error message.